### DISCUSSION TITLE
можно ли сделать Field, которые не будет пытаться сразу сохраняться в модель?

### DISCUSSION THREAD

**Discussion by pooding55** [moonshine-software/moonshine#675] [Q&A] [answered]
Можно ли добавить поля, которые не будет сразу сохраняться в модель?
Например, есть необходимость добавить поле, которое будет обрабатываться своей внутренней логикой (уйдёт в логи или что угодно), но не будет писаться в модель. Сейчас если в таблице не существует поля - получается SQLSTATE[42S22]: Column not found: 1054 Unknown column. 

Можно ли этого избежать?

**Comment by lee-to** [comment, marked as answer]
onApply

**Comment by khlystou** [comment]
Что бы избежать такой ошибки, добавляйте поля в $fillable модели, а всё остальное можно перехватить, подписавшись на событие модели или муншайн.
